src/main/views/status-description.njk

```diff
@@ -0,0 +1,68 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+{% from "./macros/common-components.njk" import goBack %}
+{{ displayAlpha }}
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ "Court Event Status" | safe }}
+{% endblock %}
+{% block beforeContent %}
+  {{ goBack('/') }}
+{% endblock %}
+{% block content %}
+  <div id="grid-row" class="govuk-grid-row">
+    <div class="layout-width-four-fifths">
+      <h2 id='page-heading' class="govuk-heading-l">Live hearing updates - glossary of terms</h2>
+      <p class="govuk-body">A list of descriptions of each hearing status that reflects in-court events</p>
+      <div class="govuk-grid-row govuk-!-margin-bottom-8">
+        {% for key, value in statusList %}
+          {% if value| length != 0 %}
+            <div class="govuk-grid-column width-column left-align-text govuk-!-font-size-19">
+              <a id='{{ key }}-selector' href='#{{ key }}' class="govuk-link govuk-link--no-visited-state">{{ key }}</a>
+            </div>
+          {% endif %}
+        {% endfor %}
+      </div>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header app-custom-class"></th>
+          <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">Hearing status</th>
+          <th scope="col" class="govuk-table__header govuk-!-width-three-quarters">Description</th>
+        </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        {% for key, value in statusList %}
+          {% for status, statusDetails in value %}
+            <tr class="govuk-table__row">
+              {% if value | length != 0 %}
+                  <th scope="row" class="govuk-table__header">
+                  {% if displayAlpha != "false" %}
+                      <h2 id="{{ key }}" class="govuk-heading-l">{{key}}</h2>
+                  {% endif %}
+                  </th>
+                <td class="govuk-table__cell">
+                  <div class="govuk-grid-column">
+                    <a id='{{ statusDetails.id }}-selector' href='#{{ statusDetails.id }}' class="link-without-formatting">{{ status }}</a>
+                  </div>
+                </td>
+                <td class="govuk-table__cell">
+                  <div><p class="govuk-body">{{ statusDetails.description }}</p></div>
+                </td>
+              {% endif %}
+            </tr>
+            {% set displayAlpha = "false" %}
+          {% endfor %}
+        {% endfor %}
+        </tbody>
+      </table>
+    </div>
+  </div>
+  <hr class="govuk-section-break--visible govuk-!-margin-top-9"></hr>
+  <p class="govuk-body"><a href="/" class="govuk-link govuk-!-font-weight-bold">Live hearing updates - select a court</a></p>
```

Comment:
Only minor, however the two links here look like they are slightly larger font than other similar links. 

e.g currently they're
<img width="444" alt="Screenshot 2021-09-21 at 09 21 38" src="https://user-images.githubusercontent.com/87066931/134137182-afafea33-e3fc-4425-b680-658d467866e3.png">

similar link: 

<img width="337" alt="Screenshot 2021-09-21 at 09 21 55" src="https://user-images.githubusercontent.com/87066931/134137219-13add80d-648e-4d1b-b946-b7c3c2d6ed9b.png">

Reply:
This has been fixed.